problems/removeVowels.js

```diff
@@ -6,8 +6,15 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+let newstr = "";
+for(let i = 0; i < str.length; i++){
+    let char = str[i]
+    if(char !== "a" && char !== "e" && char !== "i" && char !== "o" && char !== "u" && char !== "A" && char !== "E" && char !== "I" && char !== "O" && char !== "U"){
+        newstr += char
+    }
```

Comment:
instead of writing this long condition, you could instead have declared a string / array of just the vowels and checked to see if the vowels string / array has the current char included within it by using the includes method.